tenets/codelingo/code-review-comments/avoid-meaningless-package-names/codelingo.yaml

```diff
@@ -0,0 +1,39 @@
+funcs:
+  - name: isBlacklisted
+    type: asserter
+    body: |
+      function(name) {
+        var blacklist = [
+          "util",
+          "common",
+          "misc",
+          "api",
+          "types",
+          "interfaces",
+        ]
+
+        for (var i = 0; i < blacklist.length; i++) {
+          if (name === blacklist[i]) {
+            return true
+          }
+        }
```

Comment:
Do we have `blacklist.includes(name)` in our JS?

Reply:
@BlakeMScurr I believe includes() is ES6, which we do not use unfortunately :(